### DISCUSSION TITLE
Feature-rich companion app to compete Google Photos

### DISCUSSION THREAD

**Discussion by totti4ever** [photoprism/photoprism#2244] [Ideas]
Hey, I am a long-time fan of this project and love what you already achieved!

Still I am coping with fully replacing Google Photos when it comes to the women-acceptance-factor. This is especially the case with easy photo management and not 'only' presentation.

And for the ease of use this happens on the mobile. This means, moving pictures, deleting them but more than that managing albums.

When I share pictures from holiday with my wife I'm quite okay using syncthing and she is even able to add her pictures in there as well. But selecting pictures for e.g. printing an album does still require both of us to login to photoprism which is not that comfortable on a mobile.

Hope, you got the point and thanks for your great work!

**Comment by graciousgrey** [comment]
> women-acceptance-factor

Should not be underestimated in any case :D

When it comes to managing albums (https://docs.photoprism.app/user-guide/organize/albums/) what could be done better in a native app than in the current progressive web app (https://docs.photoprism.app/user-guide/pwa/)? A login would be needed in both cases.

**Comment by thedanbob** [comment]
Logins are actually my biggest pain-point with Photoprism. With a native app, you generally only have to log in once and never again unless you explicitly log out or reinstall the app. With a PWA, at least on iOS, I have to log in again on a regular basis (I'm not sure exactly how often but I'd say at least once a week). I use a password manager so I wouldn't care except that for some inane reason the iPhone doesn't make password managers available in home screen PWAs. Meaning if I find I'm logged out I have to close Photoprism, open my password manager, copy the password, reopen Photoprism, and paste it in manually.

This isn't really a criticism of Photoprism so much as a stupid and pointless limitation of iOS. But it's frustrating enough that I've looked around for an app that could serve as a front-end for Photoprism using the API, or even just webdav. Haven't found anything yet.

**Reply by graciousgrey** [reply]
Thank you very much for your feedback! 
Knowing what people miss when using a PWA will help us a lot once we find time improving it. 
The advantage of a PWA is that you have the full feature set on your phone. Developing a native app needs additional resources and it will probably always lack some features.

There is a [community maintained native app](https://github.com/thielepaul/photoprism-mobile). It is currently not compatible with our latest version. Fixing [compatibility issues](https://github.com/photoprism/photoprism/pull/995) is on our roadmap, feel free to follow the progress of this.

**Reply by thedanbob** [reply]
I'm happy to report that this issue has been resolved by iOS 17, which now supports password managers within home screen PWAs.
